### ISSUE TITLE
symset:enhanced1 seems to ignore SYMBOLS

### ISSUE THREAD

**Issue by ostrosablin** [NetHack/NetHack#786]
Probably because it goes through a new UTF8 handler, it seems that SYMBOLS options defined in nethackrc are ignored.

E.g. `SYMBOLS=S_boulder:0` that used to work with IBMgraphics just renders as default ` character for boulder when on enhanced1.

**Comment by bhaak** [comment]
Works for me, if and only if I put OPTIONS=symset:enhanced1 *after* the SYMBOLS= lines. Or leave it off completely and set the symset to enhanced1 ingame via the O menu.

**Comment by ostrosablin** [comment]
Indeed, reordering sections within nethackrc seems to help. Pretty weird, though, because logically I see it other way around (applying own changes over base symset, hence the order), but it should probably work in either order, as with other symsets.

**Comment by nhmall** [comment]
This order dependency is known and on the list of things to improve when time permits. I think it was previously reported via email, rather than through raising a GitHub issue though.

**Comment by nhmall** [comment]
This issue was opened for an earlier version of NetHack that is no longer being actively maintained. 

Please try the NetHack 5.0.0 release which does things a little differently, when it comes to customizations.
If you are still encountering an issue with this in the most-recent version, please open and describe the issue and specify the NetHack version in the opening comment.
Thank you.
